README.md

```diff
@@ -125,6 +125,10 @@ curl.request({ url: 'http://google.com', pretend: true }, function (err, stdout,
 });
 ```
 
+`curl_path` - *default: 'curl'*
+
+Use this if you want to use curlrequest in Windows or if you want to specify anothr path for curl.
```

Comment:
You've got a typo here: `anothr`. I think it's sufficient to say: *Use this to specify an alternative path for curl.*